File: src/main/scala/net/croz/scardf/query/NTripleHelper.java
package net.croz.scardf.query;

import com.hp.hpl.jena.rdf.model.Resource;
import com.hp.hpl.jena.rdf.model.impl.NTripleWriter;
import java.io.*;

class NTripleHelper extends NTripleWriter {
  public static String ntRendering( Resource r ) {
    StringWriter sw = new StringWriter();
    PrintWriter pw = new PrintWriter( sw );
    NTripleWriter.writeResource( r, pw );
    return sw.toString();
  }
}

File: src/main/scala/net/croz/scardf/query/NTripleHelper.scala
package net.croz.scardf.query

import com.hp.hpl.jena.rdf.model.Resource
import java.io.StringWriter
import java.io.PrintWriter
import com.hp.hpl.jena.rdf.model.impl.NTripleWriter

object NTripleHelper {
  def ntRendering(  r:Resource )= {
    val sw = new StringWriter()
    val pw = new PrintWriter( sw )
    NTripleWriter.writeResource( r, pw )
    sw.toString();
  }
}